src/content/docs/flutter-version.mdx

```diff
@@ -103,3 +103,12 @@ We would like Shorebird to support to as many Flutter versions as possible. If
 there are older versions of Flutter which you need Shorebird to support, please
 [let us know](https://github.com/shorebirdtech/shorebird/issues/1100).
 :::
+
+### Disabled Flutter Versions
+
+June, 6, 2024: Disabled releasing and patching for iOS 3.19.5 - 3.22.1 due to a
```

Comment:
shouldn't this be 11 since we are releasing today?

```suggestion
June, 11, 2024: Disabled releasing and patching for iOS 3.19.5 - 3.22.1 due to a
```